testing/trino-testing/src/main/java/io/trino/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -39,7 +39,7 @@
  * @see AbstractTestDistributedQueries
  */
 public abstract class AbstractTestIntegrationSmokeTest
```

Comment:
It is no longer smoke. `BaseIntegrationTest`?

Reply:
It is still smoke in my eyes